pkg/targetgroupbinding/networking_manager_test.go

```diff
@@ -1435,7 +1435,7 @@ func Test_defaultNetworkingManager_resolveEndpointSGForENI(t *testing.T) {
 		fields  fields
 		args    args
 		want    string
-		wantErr bool
+		wantErr error
```

Comment:
```suggestion
		wantErr string
```

There's little point to constructing an `error` just to unpack its message in the test.

Reply:
There's other tests in the same file that do the same so I was trying to keep it consistent.